### PR TITLE
Update Russian Translate

### PR DESCRIPTION
**Pull request by EvelRus**
Updated Russian translate

### CODE REVIEW

File: rttr-ru.po [hunk 1]
```diff
@@ -1138,11 +1145,10 @@ msgid "Don't forget to check the addon configuration!"
 msgstr "Не забудьте проверить конфигурацию дополнений!"
 
 #: libs/s25main/desktops/dskOptions.cpp:175
-msgid ""
+-msgid ""
```
Comment (Flow86) [inline rttr-ru.po]: this one is weird

**Review by Flow86 — CHANGES_REQUESTED** [review]

File: rttr-ru.po (hunk 1 above)
Comment (Flamefire) [reply rttr-ru.po]: Right, looks like a broken conflict resolution. Maybe: 
```suggestion
msgid ""
```

Comment (EvelRus) [reply rttr-ru.po]: Oops, yes... Technical error (
Is it necessary to commit and pull request again?

Comment (EvelRus) [reply rttr-ru.po]: Fixed a few more bugs, including forgetting quotes and extra spaces... Prode should be fine

Comment (Flamefire) [reply rttr-ru.po]: >  Is it necessary to commit and pull request again?

As you have likely seen adding the commits to the branch of this PR is enough. Thanks for fixing this!

File: rttr-ru.po [hunk 2]
```diff
@@ -4244,10 +4248,10 @@ msgid ""
 "You'll probably want to convert gold to iron ore, as this (on most maps)\n"
 "allows you to utilize the additional coal not needed for minting anymore."
 msgstr ""
-"Вы можете полностью удалить золото ресурсов и конвертировать их в железной руды, угля или гранита.\n"
+msgstr "Вы можете полностью удалить золотые ресурсы или превратить их в железную руду, уголь или гранит.\n"
```
Comment (Flamefire) [inline rttr-ru.po]:
```suggestion
"Вы можете полностью удалить золотые ресурсы или превратить их в железную руду, уголь или гранит.\n"
```

**Comment by Flamefire** [comment]
@EvelRus I resolved some conflicts directly in your branch. Everything looks good now, so double check, if you are happy with the changes and nothing got messed up. Then this can be merged

**Comment by EvelRus** [comment]
@Flamefire, Everything seems to be fine, thanks!

**Pull request merged**